source/renderer/app/api/api.js

```diff
@@ -534,7 +534,7 @@ export default class AdaApi {
     Logger.debug('AdaApi::calculateTransactionFee called', {
       parameters: filterLogData(request),
     });
-    const { walletId, address, amount } = request;
+    const { walletId, address, amount, walletBalance } = request;
```

Comment:
@tomothespian You can just add `availableBalance` here to make variables usage more consistent.
`const { walletId, address, amount, walletBalance, availableBalance } = request;`